### ISSUE TITLE
CI: don't build separate wheels for different Python versions

### ISSUE THREAD

**Issue by akx** [bitsandbytes-foundation/bitsandbytes#1010]
We don't need to build Python wheels with both `cp310` and `cp311` tags; Python 3.11 (and 3.12) is fine with `cp310` wheels. Since we don't actually use Python's extension ABI, that should be fine no matter what.

We just need to figure out a stable way to force always emitting a `cp310` wheel. https://github.com/karellen/wheel-axle/ seems to do that but I'm a bit hesitant to add a whole new dep for that.

**Comment by younesbelkada** [comment]
Thanks @akx ! If I understand correctly this is only for python == 3.10 & 3.11 ? For older python versions we need to build separate wheels right?

**Comment by akx** [comment]
No, a. cp38 wheel should work just as fine (though it needs a bit of experimentation).

**Comment by matthewdouglas** [comment]
Closing this as we've been building one set of wheels tagged as just `py3` for all Python versions.
